prost-build/src/code_generator.rs

```diff
@@ -184,7 +185,12 @@ impl<'a> CodeGenerator<'a> {
 
         self.append_doc();
         self.push_indent();
-        self.buf.push_str("#[derive(Clone, PartialEq, Message)]\n");
+        if cfg!(feature = "build-2018") {
+            self.buf
+                .push_str("#[derive(Clone, PartialEq, prost_derive::Message)]\n");
```

Comment:
Here and elsewhere, my understanding is it would be more robust to include a prefix `::`, e.g `::prost_derive::Message`.